UI/Source/CollectionViews/CollectionViewModelDataSource.swift

```diff
@@ -1057,10 +1062,15 @@ extension CollectionViewModelDataSource: NSCollectionViewDataSource {
 
     private func rebindViewAtIndexPath(_ indexPath: IndexPath, toViewModel viewModel: ViewModel) {
         guard let collectionView = collectionView else { return }
-        guard let item = collectionView.item(at: indexPath as IndexPath) as? CollectionViewHostItem else { return }
+        guard let item = collectionView.item(at: indexPath) as? CollectionViewHostItem else { return }
 
         willRebindViewModel(viewModel)
         item.hostedView?.bindToViewModel(viewModel)
+
+        // When rebinding, re-apply attributes so that the updated ViewModel is notified.
```

Comment:
Ug yeah, not `ViewModel` but `View`. Will fix that comment, and 👍 - we'll sync offline.